config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  
+  root :to => 'tasks#index', as: 'root'
+  
+  get '/tasks/:id/edit', to: 'tasks#edit', as: 'edit_task'
+  
+  patch '/tasks/:id', to: 'tasks#update'
```

Comment:
I assume you know this now and just forgot to remove this but these but come with the code on line 12